src/BlazorWebView/src/Maui/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Internals;
+
+[assembly: Preserve]
+
+[assembly: XmlnsDefinition("http://schemas.microsoft.com/dotnet/2021/maui", "Microsoft.AspNetCore.Components.WebView.Maui")]
```

Comment:
are we 100% sure there's no duplicate types in that namespace ? cause it'd cause tooling issues

Reply:
if there's none, go ahead and merge

Reply:
Nope, just 1 control plus a few related types. No one else uses that namespace.